Network/JsonRpc/Conduit.hs

```diff
@@ -79,7 +81,7 @@ initSession = Session <$> newTVar (IdInt 0)
 
 -- | Conduit that serializes JSON documents for sending to the network.
 encodeConduit :: (ToJSON a, Monad m) => Conduit a m ByteString
-encodeConduit = CL.map (L8.toStrict . flip L8.append "\n" . encode)
+encodeConduit = CL.map (L8.toStrict . encode)
```

Comment:
I will re-enable this since the JSONRPC standard requires each request or response in a separate line, especially when using raw sockets without headers as transport.